src/uct/base/uct_md.c

```diff
@@ -544,15 +544,12 @@ int uct_md_is_sockaddr_accessible(uct_md_h md, const ucs_sock_addr_t *sockaddr,
     return md->ops->is_sockaddr_accessible(md, sockaddr, mode);
 }
 
-uct_memory_type_owned_status_t uct_md_is_mem_type_owned(uct_md_h md,
-                                                        void *addr, size_t length)
+int uct_md_is_mem_type_owned(uct_md_h md, void *addr, size_t length)
 {
     return md->ops->is_mem_type_owned(md, addr, length);
 }
 
-uct_memory_type_owned_status_t uct_md_return_mem_type_not_detectable(uct_md_h md,
-                                                                        void *addr,
-                                                                        size_t length)
+int uct_md_return_mem_type_not_owned(uct_md_h md, void *addr, size_t length)
```

Comment:
no need to define it, can use ucs_empty_function_return_zero instead

Reply:
I defined this to avoid incompatible assignment warning because ucs_empty_function_return_zero return type is unsigned. 
I get way with warning with (void *) type casting. Updated PR.